### PR TITLE
adding contributors acknowledgement

### PR DESCRIPTION
**Pull request by kd-iqvia**
Added a list of our Contributors to recognise their work.
In the future we might consider using a bot or other solutions to give high 5s to our contributors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,3 +128,7 @@ And these have the same interface parameters as the [estimators](#estimators)
 The interface models are generally passed to `MLJBase.fit` or `MLJBase.machine`
 and integrated as part of a larger MLJ pipeline. [An example is provided](https://alan-turing-institute.github.io/MLJTutorials/end-to-end/boston-lgbm/)
 
+## Contributors ✨
+
+The list of our Contributors can be found [here](https://github.com/IQVIA-ML/LightGBM.jl/blob/master/contributorsAcknowledgment.md).
```
Comment (yalwan-iqvia) [inline README.md:133]: I think you can reduce this to `contributorsAcknowledgement.md` (relative link) and then you can previous it from github itself on your branch. Also maybe just call it CONTRIBUTORS.md

Comment (kd-iqvia) [reply README.md:133]: as for the name...it's just on TPE we also have `CONTRIBUTING.md` so I didn't want the two to be confused but here there isn't one and anyway I guess it should be fine.

Comment (kd-iqvia) [reply README.md]: I updated both the relative link and the file name plus added more contributors 😸

File: README.md [hunk 2]
```diff
@@ -128,3 +128,7 @@ And these have the same interface parameters as the [estimators](#estimators)
 The interface models are generally passed to `MLJBase.fit` or `MLJBase.machine`
 and integrated as part of a larger MLJ pipeline. [An example is provided](https://alan-turing-institute.github.io/MLJTutorials/end-to-end/boston-lgbm/)
 
+## Contributors ✨
+
+The list of our Contributors can be found [here](CONTRIBUTORS.md).
+Please don't hesitate to add yourself when you contribute.
```
Comment (yalwan-iqvia) [inline README.md:134]: 💪

**Review by yalwan-iqvia — APPROVED** [review]
Let's do it and acknowledge our contributors!

**Pull request merged**